### DISCUSSION TITLE
My Onju Build

### DISCUSSION THREAD

**Discussion by b-shaw** [justLV/onju-voice#43] [Show and tell]
I got my boards in a couple of days ago.  The first (and only) one I tested came right up and ran after the first flash.  There were a couple of issues,  I must have a different version of the Nest.  It's close,  but things don't line up particularly well - mounting holes are a bit off, touch pads don't touch anything on the board,  minor component clearance issues.  

Minor issues and after a bit of fenagling I got it to more or less fit in the package,  and it runs!

Thank you justLV for sharing this project.  I've been debating how to go about adding local voice to my Home Assistant install, and this is a sweet solution.

![20240122_112052](https://github.com/justLV/onju-voice/assets/36091448/8170dc75-dea0-46aa-a8e4-68802ecfcbfc)

![20240122_205243](https://github.com/justLV/onju-voice/assets/36091448/92fa8986-190e-4a43-8daf-0a289d706923)

![20240124_101435](https://github.com/justLV/onju-voice/assets/36091448/8766012f-1513-4768-9292-4f1a1b886405)

![20240124_101526](https://github.com/justLV/onju-voice/assets/36091448/d19b2aa4-67f3-470c-834c-7d18164bd0c2)

**Comment by b-shaw** [comment]
One more pic....

The touch contacts didn't line up with anything in my case,  so I bent the side contacts back to clear the PCB then wired the contacts to the PCB.  Not the prettiest solution but all 3 touchpads work.  

![20240124_102448](https://github.com/justLV/onju-voice/assets/36091448/eb253454-afc3-44b0-baec-d523f8e5261b)

**Comment by b-shaw** [comment]
Just figuring out - I have a Google Home Mini. :-)

That would explain a lot.  So,  this is how to fit Onju in a GHM :-)

**Reply by markwroberts0** [reply]
ahh, I'm very glad you went through this because I was wondering if this is possible (I have like 4 GHMs just sitting in drawers). Now to see if it's worth going through the hassle or if someone smarter than me will make a version that fits the GHM better...
